rcl/test/CMakeLists.txt

```diff
@@ -139,4 +146,19 @@ function(test_target_function)
   )
 endfunction()
 
+macro(connext_workaround target)
+  if(NOT "${ARGN} " STREQUAL " ")
+    message(FATAL_ERROR "append_link_flags() called with "
+      "unused arguments: ${ARGN}")
+  endif()
```

Comment:
Since this is a local macro which is not used externally I would not bother with this check and save the four lines.